demos/bio/sconnect.c

```diff
@@ -59,7 +59,7 @@ int main(int argc, char *argv[])
     if (!SSL_CTX_load_verify_locations(ssl_ctx, CAfile, NULL))
         goto err;
 
-    /* Lets make a SSL structure */
+    /* Lets make an SSL structure */
```

Comment:
(minor; I didn't spot this before) this should be `Let's` - let's fix this since we're changing this line anyway

Reply:
Updated :+1: